packages/cmrjs/tests/cmr-utils/test-cmr-utils.js

```diff
@@ -1352,3 +1357,37 @@ test('buildCMRQuery works with if the input results list is empty', (t) => {
   const actual = buildCMRQuery(results);
   t.deepEqual(actual, expected);
 });
+
+test('setECHO10Collection updates echo10 collection name and version', async (t) => {
+  const filename = 'tests/cmr-utils/data/meta.xml';
+  const cmrObject = await promisify(xml2js.parseString)(fs.readFileSync(filename, 'utf-8'), xmlParseOptions);
+  const updated = setECHO10Collection(
+    cmrObject,
+    { name: 'a', version: 'b' }
+  );
+  t.is(updated.Granule.Collection.ShortName, 'a');
+  t.is(updated.Granule.Collection.VersionId, 'b');
+});
+
+test('updateCmrFileCollections updates Echo10Files when missing', (t) => {
+  const cmrObject = {};
+  t.throws(() => setECHO10Collection(cmrObject), {
+    message: 'invalid ECHO10 cmr metadata {}, must have granule tag',
+  });
+});
+
+test('updateCmrFileCollections updates umm meta file', (t) => {
```

Comment:
It's not really updating a meta file, it's updating the passed in CMR meta object - update test title.